src/TieBreakingMarginComparator.php

```diff
@@ -35,8 +35,8 @@ public function compare(Margin $a, Margin $b) : int
             // $this->logger->info("Tie between two Margins:\n$a\n$b\n");
             echo "Tie between two Margins:\n$a\n$b\n";
             $result = $this->tieBreaker->breakTie($a, $b);
```

Comment:
While the correct winner was correclt being returned, the winners and losers were transposed in the echo output.